dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/deduplication/DeduplicationServiceIntegrationTest.java

```diff
@@ -110,6 +111,16 @@ void testAddPotentialDuplicate() throws PotentialDuplicateConflictException {
         deduplicationService.getPotentialDuplicateById(potentialDuplicate.getId()));
   }
 
+  @Test
+  void shouldFailWhenAddingAPotentialDuplicateWithInvalidUid() {
```

Comment:
is it possible to add this test somewhere more related to uid/db things? I would not expect it here

Reply:
I moved it under common, not sure it is the best place.
I accept suggestions.